src/Services/ApiService.js

```diff
@@ -11,10 +11,11 @@ const ApiService = () => {
       'Content-Type': 'application/json',
     };
 
-    const jwt = localStorage.getItem('jwt');
+    let jwt = localStorage.getItem('jwt');
     let AUTH_HEADER = {};
 
     if (jwt) {
+      jwt = jwt.slice(1, -1);
```

Comment:
@brendobrendo oops good catch! however, this actually is just the token in JSON form-- we should either _not_ JSON-ify the token when we save it to localStorage on line 10 in `auth.js` _or_ call `JSON.parse` on the jwt token here. feel free to do either